WordPress/Classes/ViewRelated/Plans/PlanListViewModel.swift

```diff
@@ -95,7 +95,7 @@ enum PlanListViewModel {
                 ImmuTableSection(
                     headerText: NSLocalizedString("WordPress.com Plans", comment: "Title for the Plans list header"),
                     rows: rows,
-                    footerText: NSLocalizedString("Manage your plan at WordPress.com/plans", comment: "Footer for Plans list"))
+                    footerText: String())
```

Comment:
Is there a reason this isn't `nil`? Makes sense to me that we can just null it out instead of creating a new empty String.